css/ckeditor-override.css

```diff
@@ -82,3 +82,147 @@ ol ol {
 .uds-tooltip-dark .fa-stack .fa-info {
   color: #191919;
 }
+<<<<<<< Updated upstream
+=======
```

Comment:
@duarte-daniela Looks like some Git conflict for some other sort of Git formatting here.